### PR TITLE
Added exports to package.json in Assets Package

### LINKED ISSUE #8503: New Assets Package is missing exports
The new `@pixi/assets` is missing the `exports` field, recently added to all other packages in v6.4.x.
https://github.com/pixijs/pixijs/blob/dev/packages/assets/package.json

Version: 6.5.0-rc.2

### PR DESCRIPTION
**Pull request by Nahdus**
##### Description of change
Added exports to Assets Package 

##### Pre-Merge Checklist

- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

fixes #8503

### CODE REVIEW

File: packages/assets/package.json [hunk 1]
```diff
@@ -14,6 +14,18 @@
   "main": "dist/cjs/assets.js",
   "module": "dist/esm/assets.js",
   "bundle": "dist/browser/assets.js",
+  "exports": {
```
Comment (bigtimebuddy) [inline packages/assets/package.json:17]:
```suggestion
  "types": "index.d.ts",
  "exports": {
```

Comment (bigtimebuddy) [reply packages/assets/package.json:17]: `types` was missing too, adding for backward compatibility

**Comment by bigtimebuddy** [comment]
Good one, than you @Nahdus!

File: packages/assets/package.json (hunk 1 above)
Comment (Nahdus) [reply packages/assets/package.json:17]: okay

**Comment by Julien-Marcou** [comment]
Isn't it missing the `"types": "index.d.ts",` ?

```diff
   "main": "dist/cjs/assets.js",
   "module": "dist/esm/assets.js",
   "bundle": "dist/browser/assets.js",
+  "types": "index.d.ts",
   "exports": {
     ".": {
       "import": {
         "types": "./index.d.ts",
         "default": "./dist/esm/assets.js"
       },
       "require": {
         "types": "./index.d.ts",
         "default": "./dist/cjs/assets.js"
       }
     }
   },
```

May be it's not necessary, but it's present in the other packages.

File: packages/assets/package.json (hunk 1 above)
Comment (bigtimebuddy) [reply packages/assets/package.json:17]: Could you commit this change please?

**Pull request merged**